### ISSUE TITLE
Ability to use a proxy

### ISSUE THREAD

**Issue by isontheline** [isontheline/pro.webssh.net#263]
**Describe the feature**
Ability to use a proxy (socks? HTTP?) in order to connect to a server.

* In general settings?
* In each connection settings?

**Comment by jarrah31** [comment]
Will this feature allow you to do proxy jumps, such as connecting directly to an ssh client via another ssh client?

The following .ssh/config example illustrates what I mean:
``` 
Host myproxypi
 HostName myurl.com
 Port 8700
 User pi

Host pisrv 
 ProxyJump myproxypi
 HostName 192.168.1.100
 User pi
 Port 22
```

Thanks.

**Comment by isontheline** [comment]
Hello @jarrah31 👋

This issue instead : https://github.com/isontheline/pro.webssh.net/issues/266
Issue #266 will let you that your illustration refers.

**Comment by jarrah31** [comment]
That's great news, thank you Arnaud!

**Comment by Angelelz** [comment]
Just bought the app! Nice job. Missing this feature to connect to my home server through a cloudflare tunnel. Thank you!

**Comment by Angelelz** [comment]
Yes, I have cloudflared in the server and in my mac and I can connect from anywhere to my home server that way. I had to set it up in the ssh config in my mac with ProxyCommand. I was hoping I could do the same with this app in my iphone.

**Comment by isontheline** [comment]
Unfortunately I need to find a “Cloudflared” Swift port code in order to reproduce the same thing on iOS
